Source/WebKit/UIProcess/BrowsingContextGroup.cpp

```diff
@@ -205,6 +206,23 @@ void BrowsingContextGroup::transitionPageToRemotePage(WebPageProxy& page, const
     set.add(WTFMove(newRemotePage));
 }
 
+void BrowsingContextGroup::transitionProvisionalPageToRemotePage(ProvisionalPageProxy& page, const WebCore::RegistrableDomain& provisionalNavigationFailureDomain)
```

Comment:
This function looks mostly the same as `transitionPageToRemotePage()`. It would be nice if they could share code.